tests/transactional/host_config.pm

```diff
@@ -40,6 +42,17 @@ sub run {
     }
 }
 
+
+sub add_staging_repos {
+    my $stage = get_required_var("STAGING");
+
+    # Add staging repository with increased priority over default ones
+    my $repo = "https://download.opensuse.org/repositories/SUSE:/ALP:/Staging:/$stage/images/repo/ALP-0.1-x86_64-Media1/";
```

Comment:
Does that mean that there are 'normal' repositories and additional 'staging repository'?

I mean, I had a problem in the past that there was no new build and the old build's repositories were gone which blocked me.

Will that be stabilised?

Reply:
I don't know about the problems in the past, but this repository was requested to be added in https://progress.opensuse.org/issues/119086#note-16. This is the expected flow for ALP staging as of now.

Reply:
@pdostal ATM, ALP doesn't do registration, so no proxy-scc mecanism can't be used. We hardcoded the "PUBLISH" repository (ie milestone repository) as default repository in the ALP image to ensure people didn't use the development repository by mistake. But for Staging, we need to use the associated staging repository instead of the PUBLISH one.